docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    data = electrodes.to_dataframe()
</pre><p>variable shouldnt be called "data" u can just call it electrodes_df</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>